data-avro-generator/src/main/java/com/linkedin/data/avro/generator/AvroSchemaGenerator.java

```diff
@@ -112,7 +113,14 @@ public static void main(String[] args) throws IOException
       System.exit(1);
     }
 
-    run(System.getProperty(GENERATOR_RESOLVER_PATH),
+    String resolverPath = System.getProperty(AbstractGenerator.GENERATOR_RESOLVER_PATH);
```

Comment:
I know I just approved, but I just realized this code is a duplicate of `RestLiToolsUtils#readArgFromFileIfNeeded`, right?

Reply:
Yes, but this is in a different module, so I didn't want to add dependency to restli-tools here.

Reply:
Makes sense